examples/step-60/doc/results.dox

```diff
@@ -0,0 +1,414 @@
+<h1>Results</h1>
+
+The first time the code is run, an exception is thrown and nothing is actually
+computed. You'll get the following output:
```

Comment:
I find both this a bit unsatisfactory. Be that as it may -- but could you maybe start by explaining why that is so, and what it means, before saying that this is what happens? It's so matter of fact. (A description of this phenomenon could also go into the introduction.)